libs/ledger-live-common/jest.config.ts

```diff
@@ -18,6 +18,10 @@ if (process.env.ONLY_INTEGRATION_TESTS) {
   testRegex = "(/__tests__/.*|(\\.|/)integration\\.(test|spec))\\.[jt]sx?$";
 }
 
+if (process.env.USE_BACKEND_MOCKS) {
+  testRegex = ["algorand/bridge.integration.test.ts"];
```

Comment:
Hardcoded to algorand?

Reply:
This is where we "enable" bridge mock support